internal/service/handlers/get_airdrop.go

```diff
@@ -13,42 +13,46 @@ import (
 
 func GetAirdrop(w http.ResponseWriter, r *http.Request) {
 	var (
-		id  = chi.URLParam(r, "id")
-		err = validation.Errors{"{id}": validation.Validate(id, validation.Required)}.Filter()
+		nullifier = chi.URLParam(r, "nullifier")
+		err       = validation.Errors{"{nullifier}": validation.Validate(nullifier, validation.Required)}.Filter()
```

Comment:
I think that this request logic shoud me separated from busniess logic. It may be small function in requests package, as we usually do. Also if this nullifier is hash or binary string may be it will be nice to validate format too